calculator/src/App.js

```diff
@@ -0,0 +1,72 @@
+import "./App.css";
+import React, { Component } from "react";
+import Buttons from "./Components/Buttons";
+import Output from "./Components/Output";
+
+class App extends Component {
+  state = {
+    result: "",
+  };
+
+  buttonPressed = (buttonName) => {
+    if (buttonName === "=") {
+      this.calculate();
+    } else if (buttonName === "=" && this.state.result === "") {
+      this.reset();
+    } else if (buttonName === "AC") {
+      this.reset();
+    } else if (buttonName === "C") {
+      this.reset();
+    } else if (buttonName === "%") {
+      this.percent();
+    } else if (buttonName === "+-") {
+      this.inverse();
+    } else
+      this.setState({
+        result: this.state.result + buttonName,
+      });
+  };
+
+  calculate = () => {
+    try {
+      this.setState({
+        result: (eval(this.state.result).toFixed(0) || "") + "",
```

Comment:
When you do change the state to include an operator you'll have to do if else statements (similar to the above function) to know which math you will have to do